app/app.go

```diff
@@ -890,6 +892,30 @@ func NewApp(
 		}
 	}
 
+	app.upgradeKeeper.SetUpgradeHandler(
+		upgradeName,
+		func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			delete(fromVM, authz.ModuleName)
+			delete(fromVM, bridgetypes.ModuleName)
+			delete(fromVM, savingstypes.ModuleName)
```

Comment:
suggestion: if the `app.upgradeKeeper.SetModuleVersionMap(ctx, app.mm.GetVersionMap())` line is removed in `InitChainer` below you shouldn't need to delete these module names.
On the current testnet, the store holds the existing module versions, so after updating to this binary these new modules will already be missing.